src/FormMigration/Steps/FormMeta.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Give\FormMigration\Steps;
+
+use Give\FormMigration\Contracts\FormMigrationStep;
+
+/**
+ * @unreleased
+ */
+class FormMeta extends FormMigrationStep
+{
+    /**
+     * @unreleased
+     */
+    public function process()
+    {
+        $oldFormId = $this->formV2->id;
+        $newFormId = $this->formV3->id;
+
+        $oldFormMeta = give()->form_meta->get_meta($oldFormId);
+
+        if ($oldFormMeta && is_array($oldFormMeta)) {
+            foreach ($oldFormMeta as $oldFormMetaKey => $oldFormMetaValue) {
+                $oldFormMetaValue = is_array($oldFormMetaValue) && count($oldFormMetaValue) === 1 ? $oldFormMetaValue[0] : $oldFormMetaValue;
```

Comment:
what is the check for `is_array` for?

Reply:
It's because the `give()->form_meta->get_meta()` returns the values as an array as you can check on the attached screenshot.

![form-meta-as-array](https://github.com/impress-org/givewp/assets/16308864/000afcc6-a023-44b4-b1e5-485a9f6a5c1a)